yarn.lock

```diff
@@ -2930,9 +2930,9 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@esbuild/android-arm64@npm:0.25.2":
-  version: 0.25.2
-  resolution: "@esbuild/android-arm64@npm:0.25.2"
+"@esbuild/android-arm64@npm:0.25.1":
+  version: 0.25.1
```

Comment:
Wonky rebase here since we are downgrading deps?

Reply:
Fixed!